### PR TITLE
Book Detail Structure

### PR DESCRIPTION
**Pull request by juanpablo-rojas**
## Summary
* Re adjusted borders, containers and some CSS values to accomplish the inVision design

## Screenshot
image

## Trello Card
https://trello.com/c/0LYyMFqW/28-detalle-de-libro-iii-estructura

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,25 +8,25 @@
...
+          <div class="description">
+            <h4 class="subtitle">Año de publicación:</h4>
+            <p class="placeholder">Nombre del año de publicación</p>
+          </div>
         </div>
       </div>
     </div>
   </body>
-</html>
+</html>
```
Comment (Nosoyleon) [inline index.html:32]: Add the empty line at the end.

File: scss/detail.scss [hunk 2]
```diff
@@ -33,13 +33,21 @@
   width: 540px;
 }
 
-.book-container {
+.main-container {
+  box-shadow: 4px 4px 10px 0 $black-05;
```
Comment (Nosoyleon) [inline scss/detail.scss:37]: This box-shadow should go in the book-container instead of here.

File: scss/detail.scss [hunk 3]
```diff
@@ -33,13 +33,21 @@
...
   display: flex;
+  height: 416px;
   margin: 24px;
+  width: 900px;
```
Comment (Nosoyleon) [inline scss/detail.scss:41]: Are you sure you need the height and width?, at this point book container is taking its content space. Also with the box-shadow in book container you will have to change the margin with padding.

File: index.html [hunk 4]
```diff
@@ -12,9 +11,7 @@
     <div class="book-container">
       <img src="images/book-cover.png" class="book-cover" alt="book-cover">
       <div class="book-detail">
-        <h1 class="title">
-          Título del libro <span class="placeholder genre">(género)</span>
-        </h1>
+        <h1 class="title">Título del libro <span class="placeholder genre">(género)</span></h1>
```
Comment (mcallegari10) [inline index.html:14]: Avoid nesting tags inside `h1` tags. For better SEO, try to keep them clean. You can use a flex display and move the `span` next to the `h1`

**Pull request merged**